readme.md

```diff
@@ -58,14 +58,13 @@ For a more in depth explanation as to why the IoT services are doing this, pleas
 
 ## Getting the SDK
 
-> Please note, for constrained device scenarios like mbed and Arduino, there are better, lighter weight SDK options available.  See [Other Azure IoT SDKs](#other-azure-iot-sdks) for more.
+> Please note, for constrained device scenarios like mbed and Arduino, there are better, lighter weight SDK options available.  See [Other Azure IoT SDKs](https://learn.microsoft.com/en-us/azure/iot-develop/concepts-using-c-sdk-and-embedded-c-sdk) to learn more.
 
 ### Packages
 
 The simplest way to get started with the Azure IoT SDKs on supported platforms is to use the following packages and libraries:
 
-- mbed:                                      [Device SDK library on MBED](./iothub_client/readme.md#mbed)
-- Arduino:                                   [Device SDK library in the Arduino IDE](./iothub_client/readme.md#arduino)
+- Arduino:                                   [Device SDK library in the Arduino IDE](https://aka.ms/arduino)
 - Windows:                                   [Device SDK on Vcpkg](./doc/setting_up_vcpkg.md#setup-c-sdk-vcpkg-for-windows-development-environment)
 - iOS:                                       [Device SDK on CocoaPod](https://cocoapods.org/pods/AzureIoTHubClient)
```

Comment:
Should we point people to a separate "Swift SDK" one to slowly phase people out of Cocoapods. AKA make it available but not the first thing we show people?

Reply:
Just saw the below addition. Should we just remove the cocoapods one?

Reply:
I've been debating that as well. Let's talk during stand-up today. I'll up for deprecating it, since we're not actively maintaining it.

Reply:
I created a different card to have that discussion. I'll merge this PR as it is.